### PR TITLE
Add integration with DatoCMS

### PR DESCRIPTION
**Pull request by delphaber**
Hi :) 

This PR adds integration with [DatoCMS](https://www.datocms.com/)

### CODE REVIEW

File: spec/solidus_content/providers/dato_cms_spec.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      object_double(
+        Dato::Site::Client.new(api_token, environment: environment).items,
+        find: fields
+      )
+    end
+    let(:fields) { instance_double(HashWithIndifferentAccess) }
+    let(:dato) { object_double(Dato::Site::Client.new(api_token, environment: environment)) }
+
+    before do
+      repo # This has to created first!
```
Comment (elia) [inline spec/solidus_content/providers/dato_cms_spec.rb]: Any reason for not using `let!(:repo)` instead? It should be equivalent from what I see.

Comment (delphaber) [reply spec/solidus_content/providers/dato_cms_spec.rb]: Years of rubocop-rspec warnings, maybe :D Hahaha! No problem, I can change it to `let!` :)

Comment (delphaber) [reply spec/solidus_content/providers/dato_cms_spec.rb]: Done!

Comment (elia) [reply spec/solidus_content/providers/dato_cms_spec.rb]: > Years of rubocop-rspec warnings

🤦‍♂️ 🤖 hahahaha 😄

**Comment by delphaber** [comment]
Don't know why tests did not run 🤔

**Comment by elia** [comment]
@delphaber once I get https://github.com/solidusio-contrib/solidus_content/pull/43 reviewed and merged they'll be fine, sorry for the inconvenience

**Comment by elia** [comment]
@delphaber done, a rebase should fix it up

**Comment by elia** [comment]
@delphaber looks like it didn't start… I just tried to disable and re-enable building from forks in CircleCI, can you try again please?

image

**Comment by delphaber** [comment]
Pushed again :) let's see!

**Comment by delphaber** [comment]
It didn't work :(

**Comment by elia** [comment]
@delphaber I've found this note in the CircleCI doc:

> Note: If a user submits a pull request to your repository from a fork, but no pipeline is triggered, then the user most likely is following a project fork on their personal account rather than the project itself of CircleCi, causing the jobs to trigger under the user’s personal account and not the organization account. To resolve this issue, have the user unfollow their fork of the project on CircleCI and instead follow the source project. This will trigger their jobs to run under the organization when they submit pull requests.

https://circleci.com/docs/2.0/oss/ is that the case?

**Comment by delphaber** [comment]
@elia I created a fresh account on CircleCI and now tests are running 🤔

I didn't know I had to do that for PRs coming from forks! Sorry about that!

**Comment by elia** [comment]
All good, at least we've found the solution 👍

**Comment by delphaber** [comment]
@elia I rebased my PR on master. All checks have passed now :)

**Review by elia — APPROVED** [review]
Finally! 🎉 Thanks, great job! 👍

**Pull request merged**